core/src/main/java/com/google/ar/sceneform/rendering/DepthTexture.java

```diff
@@ -0,0 +1,121 @@
+package com.google.ar.sceneform.rendering;
+
+import android.media.Image;
+import android.os.Handler;
+import android.os.Looper;
+
+import androidx.annotation.Nullable;
+
+import com.google.android.filament.Texture;
```

Comment:
This import or imports for each Texture can be removed.

Reply:
done